src/screens/SelectENSSheet.tsx

```diff
@@ -141,7 +145,7 @@ export default function SelectENSSheet() {
 }
 
 function ENSAvatar({ name }: { name: string }) {
-  const secondary30 = useForegroundColor('secondary30');
```

Comment:
@jxom for some reason `useForegroundColor` kept using the wrong color mode when picking the color, even when I explicitly set it using `ColorModeProvider`